packages/cactus-plugin-satp-hermes/src/main/typescript/core/stage-services/server/stage0-server-service.ts

```diff
@@ -299,6 +303,20 @@ export class Stage0ServerService extends SATPService {
 
     saveHash(sessionData, MessageType.NEW_SESSION_REQUEST, fnTag);
 
+    await storeLog(
+      {
+        sessionID: sessionData.id,
+        type: String(MessageType.NEW_SESSION_RESPONSE),
+        operation: "create",
```

Comment:
Please add exec, done, fail log operations for all methods in this service and other stages.